digitalWrite8 and wiringPiSPIxSetup in .h but not in .c

Hi,

There is 2 function i tried to use : 
digitalWrite8 and wiringPiSPIxSetup which can be found in wiringPi.h & wiringPiSPI.h but for wiringPiSPIxSetup, it is not defined anywhere and digitalWrite8 is commented inside wiringPi.c
Resulting of :
RPI_SPI.cpp:(.text+0xbd): undefined reference to `wiringPiSPIxSetup'
RPI_SPI.cpp:(.text+0x3c7): undefined reference to `digitalWrite8'

I don't know if there is an hidden reason for that or if i miss something.

Thanks in advance,

--- a/README.md
+++ b/README.md
@@ -85,8 +85,8 @@ pi@wiringdemo:~ $ gpio readall
 
 ## Documentation
 
-[German](https://github.com/WiringPi/WiringPi/blob/docu/documentation/deutsch/functions.md)   
-[English](https://github.com/WiringPi/WiringPi/blob/docu/documentation/english/functions.md)   
+[German](https://github.com/WiringPi/WiringPi/blob/master/documentation/deutsch/functions.md)   
+[English](https://github.com/WiringPi/WiringPi/blob/master/documentation/english/functions.md)   
 
 
 ## Installing

--- a/wiringPi/wiringPiSPI.c
+++ b/wiringPi/wiringPiSPI.c
@@ -1,7 +1,7 @@
 /*
  * wiringPiSPI.c:
  *	Simplified SPI access routines
- *	Copyright (c) 2012-2015 Gordon Henderson
+ *	Copyright (c) 2012-2025 Gordon Henderson and contributors
  ***********************************************************************
  * This file is part of wiringPi:
  *	https://github.com/WiringPi/WiringPi/
@@ -194,6 +194,11 @@ int wiringPiSPISetupMode (int channel, int speed, int mode) {
  *	Open the SPI device, and set it up, etc. in the default MODE 0
  *********************************************************************************
  */
+
+int wiringPiSPIxSetup (const int number, const int channel, const int speed) {
+  return wiringPiSPIxSetupMode(number, channel, speed, 0) ;
+}
+
 
 int wiringPiSPISetup (int channel, int speed) {
   return wiringPiSPIxSetupMode(0, channel, speed, 0) ;

--- a/wiringPi/wiringPiSPI.h
+++ b/wiringPi/wiringPiSPI.h
@@ -1,7 +1,7 @@
 /*
  * wiringPiSPI.h:
  *	Simplified SPI access routines
- *	Copyright (c) 2012-2024 Gordon Henderson and contributors
+ *	Copyright (c) 2012-2025 Gordon Henderson and contributors
  ***********************************************************************
  * This file is part of wiringPi:
  *	https://github.com/WiringPi/WiringPi/

